scout/server/blueprints/variant/templates/variant/components.html

```diff
@@ -191,6 +194,25 @@
   </form>
 {% endmacro %}
 
+{% macro panel_classify_sv(variant, institute, case, ACMG_OPTIONS, evaluations) %}
+  <div class="mt-3">
+    ACMG (INDEL) classification<label class="mt-3" data-bs-toggle="tooltip" title="Richards et al 2015"><a href="https://www.acmg.net/docs/standards_guidelines_for_the_interpretation_of_sequence_variants.pdf"  rel="noopener noreferrer" target="_blank" style="color: inherit;  text-decoration: inherit;">(Richards et al 2015)</a></label>.
```

Comment:
This is not probably the result you intended?

<img width="599" alt="image" src="https://github.com/user-attachments/assets/6253a0d4-95e2-4bd0-90fd-c078b5e492b6" />

Reply:
😊 Hihi, well, more like I didn't consider what was in the tooltip... The link is still good, but perhaps put the link on "classification" directly instead then.